pkg/proxy/proxy.go

```diff
@@ -123,10 +129,13 @@ func (p *Proxy) Run(stopCh <-chan struct{}) (<-chan struct{}, error) {
 		return nil, fmt.Errorf("failed to parse url: %s", err)
 	}
 
-	// set up proxy handler using proxy
+	p.handleError = p.newErrorHandler()
+
+	// Set up proxy handler using proxy
 	proxyHandler := httputil.NewSingleHostReverseProxy(url)
 	proxyHandler.Transport = p
-	proxyHandler.ErrorHandler = p.Error
+	proxyHandler.ErrorHandler = p.handleError
+	proxyHandler.FlushInterval = time.Millisecond * 50
```

Comment:
Does #137 also change/set this?

Reply:
Yes, this shouldn't really be. Want me to change it back? Doesn't really matter too much if they are all getting merged..